hack/verify-typecheck.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OPENSHIFT_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OPENSHIFT_ROOT}/hack/lib/init.sh"
+
+os::golang::verify_go_version
+
+cd "${OPENSHIFT_ROOT}"
+
+platforms=${PLATFORMS:-'linux/amd64,darwin/amd64,linux/arm,linux/386,linux/arm64,linux/ppc64le,linux/s390x,darwin/386'}
+
+go run ./vendor/k8s.io/kubernetes/test/typecheck/main.go -platform="${platforms}" "$@"
+if [[ $ret -ne 0 ]]; then
+  echo "!!! Type Check has failed. This may cause cross platform build failures." >&2
```

Comment:
Use:

```sh
  os::log::fatal "Type Check has failed. This may cause cross platform build failures.
Please see https://git.k8s.io/kubernetes/test/typecheck for more information."
fi
```

Reply:
done